bgpd/bgp_route.c

```diff
@@ -6953,6 +6953,59 @@ void bgp_purge_static_redist_routes(struct bgp *bgp)
 		bgp_purge_af_static_redist_routes(bgp, afi, safi);
 }
 
+int str2prefix_rtc(const char *str, struct prefix_rtc *p) {
+	struct ecommunity *ecom = NULL;
+	int ret;
+	int plen;
+	char *pnt;
+	char *cp;
+
+	p->prefix.origin_as = 65000;
+	p->family = AF_RTC;
+	p->prefixlen = 96;
+
+	/* Find slash inside string. */
+	pnt = strchr(str, '/');
+
+	/* String doesn't contail slash. */
+	if (pnt == NULL) {
+		ecom = ecommunity_str2com(str, ECOMMUNITY_ROUTE_TARGET, 0);
+		if (ecom == NULL) {
+		 	zlog_info("str2prefix_rtc: ecommunity_str2com failed");
+			return 0;
```

Comment:
remove zlog trace. the return value should be read by bgp_static_set_safi() that will be able to display something on the vty (vty_out)